web/docs/deployment/deployment-methods/wasp-deploy/fly.md

```diff
@@ -193,6 +193,22 @@ Fly.io offers support for both **locally** built Docker containers and **remotel
 
 If you want to build locally, supply the `--build-locally` option to `wasp deploy fly launch` or `wasp deploy fly deploy`.
 
+## Using a Custom PostgreSQL Image
+
+By default, Wasp uses the standard PostgreSQL image provided by Fly.io when creating a new database for your app. However, if your application requires specific PostgreSQL extensions (e.g., PostGIS), you can specify a custom PostgreSQL image during any of the commands that create a database.
+
+```shell
+wasp deploy fly create-db <region> --db-image <custom-postgres-image>
+wasp deploy fly setup <app-name> <region> --db-image <custom-postgres-image>
+wasp deploy fly launch <app-name> <region> --db-image <custom-postgres
```

Comment:
So this is me not knowing enough about how `wasp deploy fly` works, but what is the idea here, I need to keep repeating this argument for every command? How often do I need to repeat it, what does the workflow look like? How cumbersome / impractical is that -> it looks pretty impractical? What if they forget to pass it, what happens? If this is a problem, is there a better solution?